assets/js/pages/ChecksSelection/ChecksSelection.jsx

```diff
@@ -138,11 +136,9 @@ function ChecksSelection({
           customized={selectedCheck?.customized}
           selectedCheck={selectedCheck}
           provider={provider}
-          onClose={() => {
-            setIsCheckCustomizationModalOpen(false);
-            setSelectedCheck(null);
```

Comment:
Removing this `setSelectedCheck(null);` saves us from some flickering in the UI.
It is safe because every time we open the modal the current _selected check_ is overwritten